src/System.Management.Automation/FormatAndOutput/common/ComplexWriter.cs

```diff
@@ -416,10 +416,12 @@ private static StringCollection GenerateLinesWithoutWordWrap(DisplayCells displa
 
             for (int k = 0; k < lines.Length; k++)
             {
-                if (lines[k] == null || displayCells.Length(lines[k]) <= firstLineLen)
+                var currentLine = lines[k];
```

Comment:
```suggestion
                string currentLine = lines[k];
```

We should respect EditorConfig setting: `csharp_style_var_for_built_in_types = false`.